doc/locale/ja/LC_MESSAGES/news.po

```diff
@@ -21,12 +21,32 @@ msgstr ""
 msgid "News"
 msgstr "お知らせ"
 
-msgid "Release 9.07 - 2019-08-29"
-msgstr "9.07リリース - 2019-08-29"
+msgid "Release 9.08 - 2019-09-27"
+msgstr "9.08リリース - 2019-09-27"
 
 msgid "Improvements"
 msgstr "改良"
 
+msgid ""
+"[:doc:`/install/centos`] Added support for MariaDB 5.5.64, 10.2.27, 10.3.18, "
+"and 10.4.8"
+msgstr ""
+"[:doc:`/install/centos`] MariaDB 5.5.64、10.2.27、10.3.18、10.4.8をサポートし"
+"ました。"
+
+msgid "[:doc:`/install/centos`] Dropped 32-bit package support on CentOS 6."
+msgstr ""
+"[:doc:`/install/centos`] CentOS 6の32-bitパッケージのサポートをやめました。"
+
+msgid "Thanks"
+msgstr "感謝"
+
+msgid "kajiys"
+msgstr "kajiysさん"
+
```

Comment:
Could you add https://github.com/mroonga/mroonga/issues/274 related entry?
because it is a feedback from users.
(I've fogot to mention in commit message.

Reply:
@komainu8 finished review